assignment/api/src/java/org/sakaiproject/assignment/api/AssignmentConstants.java

```diff
@@ -45,6 +45,12 @@ public final class AssignmentConstants {
      */
     public static final String EVENT_ADD_ASSIGNMENT = "asn.new.assignment";
 
+
+    /**
+     *  SAK-44623: event for delayed assignment
```

Comment:
I wouldn't bother with the SAK. We can get that from the git history and they just end up polluting the code, imo. They seem a good idea, until everybody starts doing them :)